pyccl/core.py

```diff
@@ -174,10 +175,12 @@ class Cosmology(object):
         n_s (:obj:`float`): Primordial scalar perturbation spectral index.
         Omega_k (:obj:`float`, optional): Curvature density fraction.
             Defaults to 0.
-        N_nu_rel (:obj:`float`, optional): Effective number of massless
+        Omega_g (:obj:`float`, optional): Density in relativistic species
+            except massless neutrinos. The default of `None` corresponds
+            to setting this from the CMB temperature. Note that if a non-`None`
+            value is given, this may result in a physically inconsistent model.
```

Comment:
Please explain inconsistency -- it means inconsistency with Tcmb or not flat or inconsistency with Neff?